### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by neranguyen** [flet-dev/flet#1819] [Q&A]
Can I run Flet on a VPS? I want to create a website based on Flet

**Comment by rkleivel** [comment]
It sure can. I have a couple running inside docker containers that are deployed to VPS. But that's just one of several ways doing it
Hints:
* build a docker container from https://hub.docker.com/_/python with your code inside
* install docker on your VPS and run your container there
* point your reverse proxy to this container
* if you also need a reverse proxy, I can recommend https://hub.docker.com/r/linuxserver/swag - it also handles certificates with Let's Encrypt

Good luck
